### PR TITLE
databases-week2

### PR DESCRIPTION
**Pull request by SvitlanaBurlaka**

### CODE REVIEW

File: databases/week2/hyf-task.sql [hunk 1]
```diff
@@ -0,0 +1,75 @@
+-- Part1
+-- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO
+    task(title, description, created, updated, due_date, status_id, user_id)
+VALUES
+    ('Bake muffins', 'For 5 people', '2020-10-04 03:16:42', '2020-10-05 16:56:58', '2020-11-05 16:50:50', 2, 6);
+    
+-- Change the title of a task
+UPDATE
+    task
```
Comment (mateusms01) [inline databases/week2/hyf-task.sql]: Would suggest to keep the update [table name] on the same line.

Pointer: improves code readability and clarity

Comment (mateusms01) [reply databases/week2/hyf-task.sql]: also update other queries where this happens.

File: databases/week2/hyf-task.sql [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+WHERE id = 4;
+-- Mark a task as complete
+UPDATE 
+    task
+SET status_id = 3
+WHERE id = 5;
+-- Delete a task
+DELETE 
+FROM
+    task
```
Comment (mateusms01) [inline databases/week2/hyf-task.sql]: Unnecessary line break for both FROM and WHERE.

Pointer: improves readability and clarity

Comment (mateusms01) [reply databases/week2/hyf-task.sql]: see other occurrences and update

File: databases/week2/hyf-task.sql [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+-- Part 3
+
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT 
+    task.title, user.email
+FROM
+    task
+JOIN 
+    user
+ON task.user_id = user.id
```
Comment (mateusms01) [inline databases/week2/hyf-task.sql]: JOIN and JOIN condition are often kept on the same line. 

Pointer: improves code readability.

Comment (mateusms01) [reply databases/week2/hyf-task.sql]: please see other occurrences and update

File: databases/week2/school-database.sql [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+`name` varchar(255) NOT NULL,
+`email` varchar(255) NULL,
+`phone` varchar(255) NULL,
+`class_id` int(10) unsigned NOT NULL,
+CONSTRAINT `fk_class` FOREIGN KEY (`class_id`) REFERENCES `class` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+);
+
+-- Create an index on the name column of the student table.
+CREATE INDEX indx_class_name
+ON class(name);
```
Comment (mateusms01) [inline databases/week2/school-database.sql]: should be created on the student table. 
student(name)

File: databases/week2/students-task4-db.sql [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+`faculty_id` int(10) unsigned NOT NULL,
+CONSTRAINT `fk_faculty` FOREIGN KEY (`faculty_id`) REFERENCES `faculty` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+);
+
+CREATE TABLE `students_subjects` (
+`student_id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`subject_id` varchar(255) NOT NULL,
+`students_mark` varchar(255) NOT NULL,
+CONSTRAINT `fk_student` FOREIGN KEY (`student_id`) REFERENCES `students` (`id`) ON DELETE CASCADE ON UPDATE CASCADE,
+CONSTRAINT `fk_subject` FOREIGN KEY (`subject_id`) REFERENCES `subjects` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
```
Comment (mateusms01) [inline databases/week2/students-task4-db.sql:30]: really well defined constraints. great work

**Review by mateusms01 — CHANGES_REQUESTED** [review]
Really Good work, particularly on task 4, really well defined tables. Comments concern query formatting tips and minor changes.

File: databases/week2/hyf-task.sql (hunk 1 above)
Comment (SvitlanaBurlaka) [reply databases/week2/hyf-task.sql]: Thank you for your review!

**Pull request merged**